supervised/preprocessing/eda.py

```diff
@@ -162,3 +160,161 @@ def compute(X, y, eda_path):
             fout.close()
         except Exception as e:
             logger.error(f"There was an issue when running EDA. {str(e)}")
+
+    @staticmethod
+    def extensive_eda(X, y, save_path):
+
+        X = X.copy(deep=True)
```

Comment:
How does it work? You copy to the same variable. What is the final effect?

Reply:
Done.